orderBook.js

```diff
@@ -1,5 +1,59 @@
+/* eslint-disable indent */
 function reconcileOrder(existingBook, incomingOrder) {
-  throw new Error('Not Implemented Yet')
+
+    var updatedBook = []
+
+
+    if (existingBook.length == 0) {
+
+        updatedBook.push(incomingOrder)
+        return updatedBook
+    }
+    else if (existingBook[0]['type'] == 'sell' && incomingOrder['type'] == 'sell') {
+
+
+        updatedBook.push(existingBook[0], incomingOrder)
+
+    }
```

Comment:
I don't understand this piece. You're saying that if the first order in the book is a sell and you get a sell coming in that we should overwrite the existing sell order with the new sell order. This is gonna annoy the person who submitted the first sell order since you just dropped their order on the floor.